.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          cache: 'yarn'
+      - name: Yarn install
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: install
```

Comment:
Use of a third party action to install yarn dependencies feels unnecessary here. Could we simply call `yarn install --frozen-lockfile` instead?

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions is worth a read when using third-party actions too.